src/nvim/version.c

```diff
@@ -57,7 +57,6 @@ static char *(features[]) = {
   "+cursorbind",
   "+cursorshape",
   "+dialog_con",
-  "+diff",
```

Comment:
Actually there are plenty vim features which are non-optional in nvim, so perhaps we should do something with all such features in that list. I'm not sure that removing just that one is a good idea.